pytket/pytket/circuit/display/__init__.py

```diff
@@ -73,3 +74,20 @@ def render_circuit_jupyter(
     :param circuit: the circuit to render.
     """
     render_circuit_as_html(circuit, True)
+
+
+def view_browser(circuit: Union[Dict[str, Union[str, float, dict]], Circuit]) -> None:
```

Comment:
I see that `Union[str, float, dict]]` is used in a few places; maybe worth making a descriptive alias for it.